frontend/src/components/mine/ContactInfo/MineContactInfo.js

```diff
@@ -33,7 +33,8 @@ export class MineContactInfo extends Component {
   }
 
   handleChange = (value) => {
-    this.props.fetchParties(value);
+    const param = `?search=${value}`;
+    this.props.fetchParties(param);
```

Comment:
Why does `fetchParties` accept a query string?

It seems more idiomatic to me to pass an object to your `fetch...` methods, and have the route function handle the query string building with the `querystring` library. Like we're currently doing here:
https://github.com/bcgov/mds/blob/develop/frontend/src/constants/routes.js#L17

Thoughts?

Reply:
I followed the same Logic that `fetchMines` does, The difference here, is that the value in this particular situation doesn't come from the URL, it is the return of a `handleChange` function.

With that example, the params are all fetched from the URL and passed into `fetchMines`, The map search uses a different endpoint that accepts just the payload, not a query string and constructs the URL with the payload.
```
export const MINE_NAME_LIST = (params = {}) => `/mines/search?${queryString.stringify(params)}`;
```
In this situation, we are using the same endpoint for both so I had to create the query myself.
Does that explain my reasoning enough?

```
export const MINE_LIST_QUERY = (params) => `/mines${params}`;
export const PARTIES_LIST_QUERY = (params) => (params ? `/parties${params}` : "/parties");
```
Will likely need to change when this when the advanced filters are added anyways..

Reply:
Yep, totally. As discussed IRL, we can use a params object / refactor when implementing the advanced search 👍 

Marking as resolved